File: CS/WindowsApplication3/MyRibbonApplicationButtonInfo.cs
using System;
using System.Collections.Generic;
using System.Windows.Forms;
using DevExpress.Skins;
using DevExpress.XtraBars.Ribbon.ViewInfo;
using System.Drawing;

namespace DXSample {

    //if you need to change the ApplicationButton location, use this class
    public class MyRibbonApplicationButtonInfo : RibbonApplicationButtonInfo
    {
        public MyRibbonApplicationButtonInfo(RibbonViewInfo viewInfo) : base(viewInfo) { }

        protected MyRibbonControl Ribbon
        {
            get { return base.ViewInfo.Ribbon as MyRibbonControl; }
        }
    }
}

File: VB/WindowsApplication3/MyRibbonApplicationButtonInfo.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Collections.Generic
Imports System.Windows.Forms
Imports DevExpress.Skins
Imports DevExpress.XtraBars.Ribbon.ViewInfo
Imports System.Drawing

Namespace DXSample

	'if you need to change the ApplicationButton location, use this class
	Public Class MyRibbonApplicationButtonInfo
		Inherits RibbonApplicationButtonInfo
		Public Sub New(ByVal viewInfo As RibbonViewInfo)
			MyBase.New(viewInfo)
		End Sub

		Protected ReadOnly Property Ribbon() As MyRibbonControl
			Get
				Return TryCast(MyBase.ViewInfo.Ribbon, MyRibbonControl)
			End Get
		End Property
	End Class
End Namespace